modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/portlet.js

```diff
@@ -533,6 +582,21 @@
 				}
 			}
 
+			const activeIds = State.readKey(ACTIVE_PORTLET_IDS);
+
+			if (portletId in activeIds) {
+				console.warn(
+					`Liferay.Portlet.onLoad(): ${portletId} is already in the set of active portlet IDs`
+				);
```

Comment:
Again, I don't expect to ever see this `warn()`, but because the state is global, in theory, anybody could tamper with it, injecting whatever IDs they want. Whenever we deal with anything globally writable, we have to assume that the unexpected could happen, so, being defensive here.